src/xfile.cc

```diff
@@ -543,17 +545,12 @@ bool xbaseOpen(const char* path)
         return true;
     }
 
-    if (xbaseMakeDirectory(path) != 0) {
-        // FIXME: Leaking xbase and path.
-        return false;
-    }
-
     chdir(workingDirectory);
 
     xbase->next = gXbaseHead;
     gXbaseHead = xbase;
 
-    return true;
+    return false; // return false to trigger messages on game load
```

Comment:
This looks inconsistent. If loading of xbase is failed, should we insert it into the linked list? Or free the object instead, like in all other fail cases?